add entities user_documents, users_roles, roles, document_type

Sanek12382 opened the pull request:
users_roles не сделана так как должна представляться отношение между role и 
user user_documents не готов так как userID требует user для постоения отношения

Commit:
add entities user_documents, users_roles, roles, document_type
users_roles не сделана так как должна представляться отношение между role и user
user_documents не готов так как userID требует user для постоения отношения

Commit:
add entities user_documents, users_roles, roles, document_type
users_roles не сделана так как должна представляться отношением между role и user
user_documents не готов так как userID требует user для постоения отношения

AlekseiMaslov commented on src/main/java/com/aston/rapidride/entity/rolesAndDocs/UserDocument.java:
```diff
@@ -0,0 +1,30 @@
+    //тербуется сам user чтобы домавить отношение
+    Long userID;
+
+    private String firstName;
```
Поля не связаны с таблицей. Добавь аннотации @Column.

AlekseiMaslov reviewed (changes requested):
Проверь еще раз класс UserDocument

Commit:
add entities user_documents, users_roles, roles, document_type
users_roles не сделана так как должна представляться отношением между role и user
user_documents не готов так как userID требует user для постоения отношения
